### DISCUSSION TITLE
Welcome to jovial_svg Discussions!

### DISCUSSION THREAD

**Discussion by zathras** [zathras/jovial_svg#100] [Announcements]
##  Welcome!
I'm not quite sure how many people use Github discussions, but this seems like a reasonable thing.  Here's the Github boilerplate on what this is for.  You can:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * [You are encouraged to] welcome others and are open-minded. 

Sounds reasonable to me!  Maybe tips and whatnot on the use of `jovial_svg` would be nice?

If you want to know more about me, I have a vanity domain with a page at https://bill.jovial.com/

**Comment by Bindslev** [comment]
Great idea Zathras! I will encourage everybody who comes across this library to explore it if you have any SVG requirements. It does a lot of stuff really well. Personally I have used this library for a floorplan where users can click on a room and not only do I get the path ID of the clicked room, I can even signal it by redrawing the SVG with that particular room (or path) in another color.
I just wanted to come by here and say that this was not possible with any library I came across (not this one either) but after I suggested it here, it was swiftly implemented and I now have everything I could wish for in an SVG library! So, if you have any awesome improvements or ideas in mind, I would recommend creating an issue on it :)
